.docker/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM wordpress:5.0.3-fpm
+
+# Move WordPress files to Working Directory
+RUN mv /usr/src/wordpress /usr/src/public_html
```

Comment:
I am not in favor of doing this because it would require more non obvoius configuration, which I am not sure if we should do (atleast at this point). If you feel strongly about it though then I will open a Github issue to track it.

Reply:
What's the extra config that's needed?

On the WP side, it's just defining `WP_CONTENT_DIR` and `WP_CONTENT_URL`. It doesn't seem like it'd need any custom config in Docker, unless `wordpress:5.0.3-fpm` is some kind of special package that doesn't let you config the path or something.

Reply:
I also personally find it really annoying to have a 3rd-party external like WP mixed into the root of the site, and it can cause sometimes cause problems (like the recurring `update-root-files` fiasco on w.org).

I'm open to it if the alternatives are worse, though.

Reply:
Er, I was thinking of configuring custom content folders, not installing WP in a subfolder. What's the config that's needed? I always install WP in a subfolder, so I might have Nginx config you can just copy/paste, etc.

Reply:
Ah I see. Yeah we can move WordPress files to `/public_html/mu`. Will take care of this

Reply:
Done in https://github.com/WordPress/wordcamp.org/pull/113/commits/9b89964fb5e1eaf28b8792cd54d3f9f0a937f79b